test/test_mapper_llvm.cc

```diff
@@ -66,6 +65,176 @@ def fun(float(N, M) A, float(N, M) B) -> (C) {
   checkRtol(Cc - C, {A, B}, N * M);
 }
 
+TEST(LLVMCodegen, BasicParallel) {
+  string tc = R"TC(
+def fun(float(N, M) A, float(N, M) B) -> (C) {
+  C(i, j) = A(i, j) + B(i, j)
```

Comment:
we have TC coding style in #215 could you please update so this PR is future proof :)